platform/hale-platform.target

```diff
@@ -36,6 +36,16 @@
 		<repository location="https://download.eclipse.org/eclipse/updates/4.24/"/>
 		<unit id="org.eclipse.e4.core.tools.feature.source.feature.group" version="4.24.0.v20220420-0901"/>
 		<unit id="org.eclipse.sdk.ide" version="4.24.0.I20220607-0700"/>
+		<unit id="org.eclipse.ecf.core.feature.feature.group" version="1.6.1.v20211005-1944"/>
```

Comment:
When we played around with the hale product run configuration on @Flaminietta 's machine she was able to remove all the ECF related plugins leaving no dependency issues related to it. So maybe something that could be removed as a dependency altogether?
Did you see where dependencies to this are present? (e.g. if it is in one of our own features)